tests/parsec-cli-tests.sh

```diff
@@ -45,11 +45,13 @@ delete_key() {
 create_key() {
 # $1 - key type ("RSA" or "ECC")
 # $2 - key name
-# $3 - key usage ("SIGN" or "OAEP"), only consulted if $1 == "RSA"
+# $3 - key usage ("PKCS1_V15", "PSS" or "OAEP"), only consulted if $1 == "RSA"
```

Comment:
Be advised, [PKCS1 v1.5](https://en.wikipedia.org/wiki/PKCS_1#Schemes) is a name for both an encryption and for a signature scheme, so you'd need to disambiguate somewhere.

Reply:
Shall I rename it to `SIGN_PKCS1_V15` and `SIGN_PSS`?

Reply:
That works!